### ISSUE TITLE
[BUG]  Web App - Unable to Sign In with phone or email

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5151]
**Describe the bug**
Web App users are unable to sign in. Please see attached video

**To Reproduce**
1. Using Incognito browser (to enter orientation screens)
2. Go to app.illinois.edu
3. Click through Orientation screens until Sign in option
4. Click Sign in

**Expected behavior**
User should be able to enter email or phone number to login. 

**Actual behavior**
User is unable to type any text or numerical values into the field. 

**Screenshots**

https://github.com/user-attachments/assets/2817e901-518e-4f95-a3f5-bec1f84a9968

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**

**Comment by acovher2** [comment]
Unable to reproduce, perhaps Shibboleth failed? Can you please let me know what you think?
Thank you,
Amanda

**Comment by acovher2** [comment]
@vburgett I'm not sure I follow. I thought all users were using app.illinois.edu..... This is not an admin website..... Please help me understand further. Plus the nature of this defect is that the page should NEVER have gone to the phone and email sign it, I selected NETID sign in.

**Comment by acovher2** [comment]
@vburgett Thank you, that helps. So, effort not wasted, but also not relevant to the bug at hand since this is a bug about why this screen came up at all in the first place and why user can't enter inputs when it does. I think that it may be closeable if unable to reproduce. I wanted Mark to chime in about that but also to be aware of this anomaly happening too.

**Comment by acovher2** [comment]
@pmarkhennessy This bug is not about who can sign in, but rather about WHY this screen came up instead of NETID login as selected. Please advise.

**Comment by vburgett** [comment]
@acovher2 I only am directed to the sign-in page with phone number or alternate email when I don't select student or employee under the "Who are you" onboarding screen. This behavior is as expected.

Using the website (7.2.22, app.illinois.edu), I can now type into the phone number or email field you refer to in this request. Can you retest? Thank you!
